samples/matter/template/CMakeLists.txt

```diff
@@ -49,10 +49,15 @@ target_sources(app PRIVATE
     ${COMMON_ROOT}/src/app/matter_event_handler.cpp
 )
 
-if(CONFIG_CHIP_OTA_REQUESTOR)
+if(CONFIG_CHIP_OTA_REQUESTOR OR CONFIG_MCUMGR_TRANSPORT_BT)
     target_sources(app PRIVATE ${COMMON_ROOT}/src/dfu/ota/ota_util.cpp)
 endif()
 
+if(CONFIG_MCUMGR_TRANSPORT_BT)
```

Comment:
Will the SMP DFU will be enabled while clicking the BUTTON 1?

Reply:
It will start automatically if the CONFIG_CHIP_DFU_OVER_BT_SMP config is set. It works this way because there is CONFIG_CHIP_ENABLE_PAIRING_AUTOSTART=y. So clicking the BUTTON 1 doesn't change anything :)

Reply:
So if the Matter BLE advertisement is disabled e.g. the device is already commissioned?
Are we able to trigger advertising DFU SMP ?

Reply:
@ArekBalysNordic it is possible to trigger DFU SMP advertisement after reset while the device is commissioned?